.circleci/config.yml

```diff
@@ -974,6 +974,132 @@ jobs:
                     GO: /usr/lib/go/bin/go
             - store_artifacts:
                   path: test_output/artifacts
+    test--dev--coda-peers-test:
```

Comment:
are these changes already in the .jinja file?

Reply:
The changes that make this happen are in #5292